specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/examples/GetWorkloadClassifier.json

```diff
@@ -0,0 +1,28 @@
+﻿{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "Default-SQL-SouthEastAsia",
+    "serverName": "testsvr",
+    "databaseName": "testdb",
+    "workloadGroupName": "wlm_workloadgroup",
+    "workloadClassifierName": "wlm_classifier",
+    "api-version": "2019-06-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "properties": {
+          "memberName": "dbo",
+          "label": "test_label",
+          "context": "test_context",
+          "startTime": "12:00",
+          "endTime": "14:00",
+          "importance": "high"
+        },
+        "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/Default-SQL-SouthEastAsia/providers/Microsoft.Sql/servers/testsvr/databases/testdb/workloadGroups/smallrc",
```

Comment:
Across the samples, the classifier resource ids and types should not be for: Microsoft.Sql/servers/databases/dataWarehouseWorkloadGroups

Reply:
Thanks! That was a good catch. Addressed in latest commits